syntax = "proto3";

package fixbug;

option cc_generic_services = true;

message ResultCode{
    int32 errCode = 1;
    bytes errMsg = 2;
}

message GetFriendsListRequest{
    uint32 userId = 1;
}

message GetFriendsListResponse{
    ResultCode result = 1;
    repeated bytes friends = 2;
}

service FriendServiceRpc{
    rpc GetFriendsList(GetFriendsListRequest) returns(GetFriendsListResponse);
}
